src/commandHandlers/codeOfConduct.ts

```diff
@@ -24,7 +24,7 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
     );
 };
 
-export const description = 'Code Of Conduct';
```

Comment:
I'm not sure if this needs to be lower-cased.
Most of the times, people abbreviate this as COC, and I think, semantically it might be more consistent to use `Of`
Though, not sure which one is better.

Thanks for opening this PR!

Reply:
Yeah, it makes sense. The reason why I proposed this change was for consistency matters. In every other place in the project, the title "Code **o**f Conduct" is written with the "**o**" in lower case.